.ebextensions/db.config

```diff
@@ -6,7 +6,7 @@ commands:
         'https://yum.postgresql.org/11/redhat/rhel-6-x86_64/postgresql11-libs-11.5-1PGDG.rhel6.x86_64.rpm' \
         'https://yum.postgresql.org/11/redhat/rhel-6-x86_64/postgresql11-11.5-1PGDG.rhel6.x86_64.rpm' \
         'https://yum.postgresql.org/11/redhat/rhel-6-x86_64/postgresql11-devel-11.5-1PGDG.rhel6.x86_64.rpm'
-      /bin/ln -sf /usr/pgsql-11/bin/pg_config /usr/local/bin/pg_config
+      /bin/ln -s /usr/pgsql-11/bin/pg_config /usr/local/bin/pg_config
```

Comment:
I changed this back to `-s` @matschaffer as this link should never have previously existed in the new image that EB is using and I would like it to fail if it does exist for some reason -- let me know if I missed something here.

Reply:
Yeah, pretty sure it needs -f otherwise it’ll fail on the next deploy when it tries to recreate. At least I ran into that while I was testing.